CMakeLists.txt

```diff
@@ -216,10 +216,18 @@ target_include_directories(celerity_runtime PUBLIC
   $<INSTALL_INTERFACE:include/celerity/vendor>
 )
 
+if(WIN32 AND CELERITY_SYCL_IMPL STREQUAL "DPC++")
```

Comment:
Maybe you could add a note here as to why this is even needed (most likely a bug in DPC++, right?)

Reply:
I'm not sure if it's a bug in DPC++. Linking against the correct debug/release library is a windows thing, because debug/release are not ABI compatible on windows. Maybe DPC++ could do this automatically with cmake, but I don't know if that's the intended behavior.

But I can of course add a comment regardless.